src/main/java/de/hpi/bpt/argos/util/ObjectWrapper.java

```diff
@@ -0,0 +1,26 @@
+package de.hpi.bpt.argos.util;
+
+/**
+ * This object is used to wrap another object.
+ * @param <ObjectType> - the type of the object to wrap
+ */
+public class ObjectWrapper<ObjectType> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00119)